.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: go
 
+go:
+  - 1.12.x
```

Comment:
why bump the version? Is it required because of go mod? If so, please move this change to the "go mod" PR, instead of this one. This PR is supposed to fix the test.sh bug.